build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python src/etc/build_rust.py $@
```

Comment:
I’d just call python2 here, honestly, unless you’re absolutely sure it will work on both python3 and python2.